docs/source/tutorial.rst

```diff
@@ -174,6 +174,26 @@ the dataset we generated earlier:
 
     engine.fit(dataset)
 
+Note that by default, the training of the engine is non-deterministic: if you
+train your NLU twice on the same data and test it on the same input, you'll get
+different outputs.
+
+If you want to run training in a reproducible way you can pass a random seed to
+your engine:
```

Comment:
I prefer using a more impersonal form in the documentation, but that's just a suggestion. That would be something like:
```
Reproducible training and testing can be achieved by passing a 
**random seed** to the engine:
```